src/app/public/index.html

```diff
@@ -32,6 +32,10 @@
     <title>Open Apparel Registry (beta)</title>
     <!-- Environment Variables -->
     <script src="%PUBLIC_URL%/web/environment.js"></script>
+    <!-- TODO: Replace with actual Google Tag Manager snippet -->
```

Comment:
Going on my first comment, maybe we should have some check to see if `REACT_APP_GOOGLE_ANALYTICS_KEY` is `null`, and if it is, we shouldn't inject anything into the page. 

Is something like this possible?

Reply:
Yes. I added a demo of that in 0ef53f2.